include/cantera/transport/GasTransport.h

```diff
@@ -25,53 +25,35 @@ class MMCollisionInt;
 class GasTransport : public Transport
 {
 public:
-    //! Viscosity of the mixture  (kg /m /s)
+    //! Get the viscosity [Pa·s] of the mixture
     /*!
-     * The viscosity is computed using the Wilke mixture rule (kg /m /s)
+     * The viscosity is computed using the Wilke mixture rule:
      *
      * @f[
-     *     \mu = \sum_k \frac{\mu_k X_k}{\sum_j \Phi_{k,j} X_j}.
+     *     \eta = \sum_k \frac{\eta_k X_k}{\sum_j \Phi_{k,j} X_j}.
      * @f]
      *
-     * Here @f$ \mu_k @f$ is the viscosity of pure species @e k, and
-     *
+     * Here @f$ \eta_k @f$ is the viscosity of pure species *k*. The formula for the
+     * weighting function is given in Poling et al. @cite poling2001, Eq. (9-5.14):
      * @f[
-     *     \Phi_{k,j} = \frac{\left[1
-     *                  + \sqrt{\left(\frac{\mu_k}{\mu_j}\sqrt{\frac{M_j}{M_k}}\right)}\right]^2}
-     *                  {\sqrt{8}\sqrt{1 + M_k/M_j}}
+     *     \Phi_{k,j} = \frac{ \left[ 1 + \left( \eta_k / \eta_j \right)^{1/2}
+     *                               \left( M_j / M_k \right)^{1/4} \right]^2 }
+     *                      {\left[ 8 \left( 1 + M_k / M_j \right) \right]^{1/2}}
      * @f]
-     *
-     * @returns the viscosity of the mixture (units =  Pa s = kg /m /s)
-     *
      * @see updateViscosity_T()
      */
     double viscosity() override;
 
-    //! Get the pure-species viscosities
+    //! Get the pure-species viscosities [Pa·s]
     void getSpeciesViscosities(double* const visc) override {
         update_T();
         updateViscosity_T();
         std::copy(m_visc.begin(), m_visc.end(), visc);
     }
 
-    /**
```

Comment:
Should this docstring be deleted? It is an override of a class defined in `Transport.h`, which is not implemented and has a significantly shorter docstring.

Reply:
This docstring for `getBinaryDiffCoeffs` is longer but mostly redundant (and wrong in multiple ways). The updated docs for the base class method correctly and fully describes the interface.